oecophylla/taxonomy/taxonomy.rule

```diff
@@ -78,17 +83,23 @@ rule metaphlan2:
 
 rule taxonomy_kraken:
     """
-    Runs Kraken using general defaults and keeps mapping file.
+    Runs Kraken with Bracken to construct taxonomic profiles.
     """
     input:
         forward = qc_dir + "{sample}/filtered/{sample}.R1.trimmed.filtered.fastq.gz",
         reverse = qc_dir + "{sample}/filtered/{sample}.R2.trimmed.filtered.fastq.gz"
     output:
-        map = taxonomy_dir + "{sample}/kraken/{sample}.map.txt",
         report = taxonomy_dir + "{sample}/kraken/{sample}.report.txt",
+        profile = taxonomy_dir + "{sample}/kraken/{sample}.profile.txt"
+        # extra output files:
```

Comment:
Per earlier comment in person, would prefer to have this documentation in triple quotes description at top of rule rather than single comment lines here. It looks too much like a temporary commented out valid output specification here.

Reply:
How about I make it triple quotes in place (instead of rule top)?